packages/x-charts/src/SparkLineChart/SparkLineChart.tsx

```diff
@@ -201,12 +191,14 @@ const SparkLineChart = React.forwardRef(function SparkLineChart(
           data: Array.from({ length: data.length }, (_, index) => index),
           hideTooltip: xAxis === undefined,
           ...xAxis,
+          position: 'none',
```

Comment:
@mui/xcharts one possible disadvantage of the current approach is that the axis is being counted for sizing purposes even if `ChartsAxis` is not present in the document.

Since we have "defaults", I guess it would be difficult to prevent this kind of behavior :/